authors/apps/authentication/views.py

```diff
@@ -44,7 +51,20 @@ def post(self, request):
         serializer = self.serializer_class(data=user)
         serializer.is_valid(raise_exception=True)
 
-        return Response(serializer.data, status=status.HTTP_200_OK)
+        token = generate_token(serializer.data).decode()
```

Comment:
I noticed that the token generated for login and registration are the same. If you include a payload with details such as the time the token is generated, it will generate different tokens for each. You can also include expiry inside it.
` payload = {
            'exp': datetime.datetime.utcnow() + datetime.timedelta(days=0, seconds=5),
            'iat': datetime.datetime.utcnow(),
            'sub': output
        }`

Reply:
Great feedback. 👏 @RuthNjeri

Reply:
Okay, I will put the issuing time and expiry time in the payload to be encoded by 9.30 a.m.
Thank you @RuthNjeri